content/en/integrations/faq/aws-integration-and-cloudwatch-faq.md

```diff
@@ -15,22 +17,37 @@ Other APIs are used to enrich metrics data. Some examples include:
+ * Gathering information about the status / health of resources, such as automuting
 
- * Gathering log streams for processing into metrics
+ * Gathering log streams
 
-### How many API requests will be made? How can I monitor my CloudWatch usage?
+### How many API requests are made and how can I monitor my CloudWatch usage?
 
-Datadog gathers the available metrics every 10 minutes for each sub-integration you have installed from a tile or the checklist on the main tile. If you have a large number of AWS resources for a particular sub-integration (e.g. SQS, ELB, DynamoDB, AWS Custom metrics), this can impact your AWS CloudWatch bill.
+Datadog gathers the available metrics every 10 minutes for each AWS sub-integration you have installed. If you have a large number of AWS resources for a particular sub-integration (SQS, ELB, DynamoDB, AWS Custom metrics), this can impact your AWS CloudWatch bill.
 
-You can monitor your CloudWatch API usage using the [AWS Billing integration][2]. For more information, see [Using Datadog's AWS Billing Integration to monitor your CloudWatch usage][3].
+You can monitor your CloudWatch API usage using the [AWS Billing integration][2].
 
-### How can I reduce the lag on my CloudWatch metrics to appear in Datadog/Why are my CloudWatch metrics delayed?
+### How can I reduce the delay of receiving my CloudWatch metrics to Datadog?
 
-By default, Datadog collects AWS metrics every 10 minutes. See [Cloud Metric Delay][4] for more information. If you need to reduce the latency, contact [Datadog support][5] for assistance.
+By default, Datadog collects AWS metrics every 10 minutes. See [Cloud Metric Delay][3] for more information. If you need to reduce the latency, contact [Datadog support][4] for assistance.
+
+### Is there a discrepancy between my data in CloudWatch and Datadog?
+
+There are two important distinctions to be aware of:
+
+1. In AWS for counters, a graph set to `sum` `1 minute` shows the total number of occurrences in one minute leading up to that point (the rate per 1 minute). Datadog is displaying the raw data from AWS normalized to per second values, regardless of the timeframe selected in AWS. Therefore, you might see a lower value in Datadog.
+2. Overall, min, max, and avg have different meanings within AWS. AWS distinctly collects average latency, minimum latency, and maximum latency. When pulling metrics from AWS Cloudwatch, Datadog only receives the average latency as a single timeseries per ELB. Within Datadog, when you select `min`, `max`, or `avg`, you are controlling how multiple timeseries are combined. For example, requesting `system.cpu.idle` without any filter returns one series for each host reporting that metric. Datadog combines these time series using [space aggregation][5]. Otherwise, if you requested `system.cpu.idle` from a single host, no aggregation is necessary and switching between avg and max yields the same result.
```

Comment:
```suggestion
2. Overall, `min`, `max`, and `avg` have different meanings within AWS. AWS distinctly collects average latency, minimum latency, and maximum latency. When pulling metrics from AWS CloudWatch, Datadog only receives the average latency as a single timeseries per ELB. Within Datadog, when you select `min`, `max`, or `avg`, you are controlling how multiple timeseries are combined. For example, requesting `system.cpu.idle` without any filter returns one series for each host reporting that metric. Datadog combines these time series using [space aggregation][5]. Otherwise, if you requested `system.cpu.idle` from a single host, no aggregation is necessary and switching between `avg` and `max` yields the same result.
```